autogpt/prompts/default_prompts.py

```diff
@@ -0,0 +1,29 @@
+SYSTEM_PROMPT_AICONFIG_AUTOMATIC = ("""
+    Your task is to devise up to 5 highly effective goals and an appropriate role-based name (_GPT) for an autonomous agent, ensuring that the goals are optimally aligned with the successful completion of its assigned task.
+
+    The user will provide the task, you will provide only the output in the exact format specified below with no explanation or conversation.
+
+    Example input:
+    Help me with marketing my business
+
+    Example output:
+    Name: CMOGPT
+    Description: a professional digital marketer AI that assists Solopreneurs in growing their businesses by providing world-class expertise in solving marketing problems for SaaS, content products, agencies, and more.
+    Goals:
+    - Engage in effective problem-solving, prioritization, planning, and supporting execution to address your marketing needs as your virtual Chief Marketing Officer.
+
+    - Provide specific, actionable, and concise advice to help you make informed decisions without the use of platitudes or overly wordy explanations.
+
+    - Identify and prioritize quick wins and cost-effective campaigns that maximize results with minimal time and budget investment.
+
+    - Proactively take the lead in guiding you and offering suggestions when faced with unclear information or uncertainty to ensure your marketing strategy remains on track.
+    """)
+
+TASK_PROMPT_AICONFIG_AUTOMATIC = (
+    "Task: '{{user_prompt}}'\n"
+    "Respond only with the output in the exact format specified in the system prompt, with no explanation or conversation.\n"
+)
+
+USER_DESIRE_DEFAULT_PROMOTE = (
+    "Write a wikipedia style article about the project: https://github.com/significant-gravitas/Auto-GPT"
+)  # Default prompt
```

Comment:
Create categories so people know where the constant is coming from.

Here it's 
#########################Setup.py#################################